agent/__init__.py

```diff
@@ -642,6 +648,29 @@ def write_metadata(data, rewrite_file):
     metadata_path.chmod(0o644)
 
 
+def enroll_device(enroll_token, claim_token, device_id):
+
```

Comment:
pls stop:
1) adding unnecessary blank lines
2) spread quite obvious/simple/short code over multiple lines if it can fit one line

Reply:
The first of all (right after good working) code must be lite readable. 
And thus i have a habit to leave blank line after function header, just for regular form. Because some functions have a long args list which is not fits on one line. 
The next one is a dictionaries, with strings, when it wrote line by line this is reading simply than if this distributed on several lines by 3-5 non regular items per line. 
And third one,  yes i have to place space lines between logical or data blocks. I do not understand you. Did you like a glued text? 

Ok then, i could do that way. But this is not good idea in 30-50% of cases.
Fixed.

Reply:
> And one question left.
> Now it checks enrollment key record only at bootstraping.
> May be we need to check it somwhere else? Or there is claim url for other cases?

nothing comes to my mind now, but it's your task to check the whole project's codebase for finding an answer

Reply:
This is more behavior question than code.  @vpetersson do we need to check for enroll token in config any time else than on bootstrap install?  I assume not, but may be i'm wrong?

Reply:
That's a good question. I guess we should check it if the device is unclaimed.

Reply:
Ok. Added check/enroll to "main cycle".